com.unity.render-pipelines.universal/Shaders/Nature/SpeedTreeUtility.hlsl

```diff
@@ -0,0 +1,34 @@
+#ifndef UNIVERSAL_SPEEDTREE_UTILITY
+#define UNIVERSAL_SPEEDTREE_UTILITY
+
+uint2 ComputeFadeMaskSeed(float3 V, uint2 positionSS)
```

Comment:
I guess this function comes from the graphics package?

Reply:
A version of it exists inside HDRP, although it uses a different variable name. Its goal is to prevent camera rotation causing large differences in the dither pattern. You can still see some sparkly noise, but it prevents obvious scrolling when a player looks around, for example.

Reply:
If there is no way to merge this with the HDRP one and put it into Core SRP, I would suggest at least putting it into a common URP shader include, since to me this seems like a useful function for crossfade dither on any object, not only speed trees?

Reply:
Do you know if there is an existing utility include file in URP where this would naturally fit? Perhaps something in Shaders/Utils where it's well suited?
The logic is wrapped behind #if defined(SHADER_QUALITY_HIGH). Does that sound reasonable as a common quality requirement (high quality) if we want to expose this generally?